### ISSUE TITLE
[API] Typo in parameter list for fused sampling kernel (top_k_top_p_sampling_from_logits)

### ISSUE THREAD

**Issue by kasohrab** [flashinfer-ai/flashinfer#873]
There appears to be a param name mistake in the API [for sampling.top_k_top_p_sampling_from_logits](https://github.com/flashinfer-ai/flashinfer/blob/78dde79bffd51cb862bd2c553bb4a3756902ebd4/flashinfer/sampling.py#L706). It should be logits instead of probs here right? That's what the docs below say too at least.

**Comment by yzh119** [comment]
Hi @kasohrab yes are you right! can you create a PR to fix that?

**Comment by kasohrab** [comment]
@yzh119 Done please take a look when you have a chance -- thanks for this project!
